server/room.ts

```diff
@@ -79,14 +79,22 @@ export class Room {
       // console.log(this.roomId, this.password, password);
       if (postgres) {
         const result = await postgres.query(
-          `SELECT password FROM room where roomId = $1`,
+          `SELECT password, isSubRoom FROM room where roomId = $1`,
```

Comment:
should probably match cases here so it's not confusing. Use camel_case if needed?